common/app/views/fragments/containers/facia_cards/standardHeaderMeta.scala.html

```diff
@@ -7,7 +7,9 @@
 @import model.Badges.badgeFor
 
 @defining((containerDefinition.displayName, containerDefinition.href)) { case (title, href) =>
-    <div class="fc-container__header  @if(containerDefinition.customHeader.isEmpty) {js-container__header}">
+    <div class="fc-container__header
+                @if(containerDefinition.customHeader.isEmpty) {js-container__header}
+                @badgeFor(containerDefinition).map { badge => fc-container__header--@badge.classModifier}">
```

Comment:
Is every badge guaranteed to have a modifier? It might be better to make this an `Option[String]` and have a default if one isn't provided.